Tensile/Common.py

```diff
@@ -704,6 +717,14 @@
     "LdsPadA":                     [ -1, 0, 1, 2, 3, 4, 8],
     "LdsPadB":                     [ -1, 0, 1, 2, 3, 4, 8],
 
+    # Padding boundary for LDS. defines block-size for pad insertion. for every 'LdsBlockSizePerPad' bytes, LDS padding (pad value from LdsPad parameter)
+    # is added (readOffset aware of the pad and adjusts offset value based on this parameter value).good rule of thumb is LdsBlockSizePerPad >= unrollDepth * BPE
+    # optimized value is 128
+    "LdsBlockSizePerPad":          [-1, 64, 128, 256],
```

Comment:
Does -1 mean no padding?

Reply:
-1 ; design choose value

Reply:
how to disable "LdsBlockSizePerPad" concept to use legacy per unroll padding?

Reply:
by default LdsBlockSizePerPad = -1;  when TransposeLds=1 LdsBlockSizePerPad set to 256..